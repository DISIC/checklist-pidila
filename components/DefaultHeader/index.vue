<template src="../../data/layout/header.html"/>

<script>
  export default {
    name: 'DefaultHeader'
  }
</script>

<style lang="css" scoped>
  @import "../../data/layout/layout-override.css";
</style>
