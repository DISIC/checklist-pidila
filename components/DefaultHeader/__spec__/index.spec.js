import {shallowMount} from '@vue/test-utils'

import DefaultHeader from '../index.vue'

describe('components | DefaultHeader', () => {

  it('should match snapshot', () => {
    // when
    const wrapper = shallowMount(DefaultHeader)

    // then
    expect(wrapper.element).toMatchSnapshot()
  })

})
