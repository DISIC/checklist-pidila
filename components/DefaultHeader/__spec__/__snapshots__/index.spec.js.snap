// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | DefaultHeader should match snapshot 1`] = `
<header
  class="banner default-banner"
  role="banner"
>
  <div
    class="container"
  >
    <p
      class="site-title"
    >
      Checklist Pidila
    </p>
     
    <p
      class="baseline"
    >
      
      Une liste unique des bonnes pratiques pour les sites publics : 
      <abbr
        title="Référentiel général d’amélioration de l’accessibilité"
      >
        RGAA
      </abbr>
      , Charte internet de l'État, Cerfa numérique, Bonnes pratiques Opquast, Éco-conception
    
    </p>
  </div>
</header>
`;
